packages/react/src/components/Modal/Modal.stories.tsx

```diff
@@ -175,3 +177,93 @@ export const ModalWithMaxWidth: StoryFn<typeof Modal> = () => {
     </>
   );
 };
+
+export const defaultArgs = {
+  label: 'Velg et fylke',
+  multiple: false,
+  options: [
+    {
+      label: 'Agder',
+      value: 'Agd',
+      keywords: ['agder', 'aust-agder', 'vest-agder'],
+    },
+    {
+      label: 'Innlandet',
+      value: 'Inn',
+      keywords: ['innlandet', 'hedmark', 'oppland'],
+    },
+    {
+      label: 'Møre og Romsdal',
+      value: 'MoR',
+      keywords: ['møre og romsdal', 'møre', 'romsdal'],
+    },
+    { label: 'Nordland', value: 'Nor', keywords: ['nordland'] },
+    { label: 'Oslo', value: 'Osl', keywords: ['oslo'] },
+    { label: 'Rogaland', value: 'Rog', keywords: ['rogaland'] },
+    {
+      label: 'Vestfold og Telemark',
+      value: 'VoT',
+      keywords: ['vestfold og telemark', 'vestfold', 'telemark'],
+    },
+    {
+      label: 'Troms og Finnmark',
+      value: 'ToF',
+      keywords: ['troms og finnmark', 'troms', 'finnmark'],
+    },
+    {
+      label: 'Trøndelag',
+      value: 'Trø',
+      keywords: ['trøndelag', 'nord-trøndelag', 'sør-trøndelag'],
+    },
+    {
+      label: 'Vestland',
+      value: 'Ves',
+      keywords: ['vestland', 'hordaland', 'sogn og fjordane'],
+    },
+    {
+      label: 'Viken',
+      value: 'Vik',
+      keywords: ['viken', 'østfold', 'akershus', 'buskerud'],
+    },
+  ],
+};
+
+export const ModalWithSelect: StoryFn<typeof Modal> = () => {
+  const modalRef = useRef<HTMLDialogElement>(null);
+  const [input, setInput] = useState('');
+
+  return (
+    <>
+      <Button onClick={() => modalRef.current?.showModal()}>Open Modal</Button>
+      <Modal
+        ref={modalRef}
+        onClose={() => setInput('')}
+        style={{ overflow: 'visible' }}
+      >
+        <Modal.Header>Modal med skjema</Modal.Header>
+        <Modal.Content>
+          <Select
+            portal={false}
+            {...{ ...defaultArgs }}
+          ></Select>
+        </Modal.Content>
+        <Modal.Footer>
+          <Button
+            onClick={() => {
+              window.alert(`Du har sendt inn skjema med navn: ${input}`);
+              modalRef.current?.close();
+            }}
+          >
+            Send inn skjema
+          </Button>
+          <Button
+            variant='secondary'
+            onClick={() => modalRef.current?.close()}
+          >
+            Avbryt
+          </Button>
+        </Modal.Footer>
+      </Modal>
+    </>
+  );
+};
```

Comment:
Remove the form aspect of this modal, since `input` is never set 😄 
![image](https://github.com/digdir/designsystem/assets/14057205/6bee08a0-c27a-47b4-8bca-8a4dd823328b)

Reply:
Woops, done